### PR TITLE
[IN_PROGRESS]app: Nuke: ask user to update when script range is locked (#9912)

### PR DESCRIPTION
**Pull request by JeanChristopheMorinRodeoFX**
[[Ticket #9912](https://shotgun1.rodeofx.com/page/264026#Ticket_9912_Nuke%20framerange%20has%20a%20mind%20of%20its%20own)]

When Nuke script range is locked, prompt the user to choose to update or not the frame range.

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -166,17 +171,28 @@ def set_frame_range(self, engine, in_frame, out_frame):
...
             locked = nuke.root()["lock_range"].value()
+            sync = False
             if locked:
+                sync = QtGui.QMessageBox.question(None,
+                                                  "Script range is locked",
+                                                  ("Script range is locked. Do you want to "
+                                                   "sync frame range from Shotgun?"),
+                                                  QtGui.QMessageBox.Yes | QtGui.QMessageBox.No)
+
+            if sync == QtGui.QMessageBox.Yes:
```
Comment (kookiCT) [inline app.py]: The range will never be updated if the range isn't locked with that test !

**Comment by JeanChristopheMorinRodeoFX** [comment]
@kookiCT I fixed your comment (and I don't know why, but your comment disappeared).

**Comment by JeanChristopheMorinRodeoFX** [comment]
Also rebased with latest changes from the update

**Comment by JeanChristopheMorinRodeoFX** [comment]
Testing comments from GitHub to GitColony

**Pull request closed without merging**